timeless/auth/auth.py

```diff
@@ -6,13 +6,21 @@
 from flask import session
 from passlib.handlers.bcrypt import bcrypt_sha256
 
+from timeless import DB
 from timeless.employees.models import Employee
 from timeless.mail import MAIL
 from flask_mail import Message
 
 
 PASS_LENGTH = 8
 
+
+def mask_email(email: str) -> str:
+    """ It masks provided email with asteriks """
+    username, host = email.split("@")
+    return username[:3] + (len(username) - 3) * "*" + f"@{host}"
```

Comment:
@vryazanov what does ``f"@{host}"`` do exactly? (really strange syntax) :D

Reply:
@vryazanov one more thing: what happens if the username has less then 3 characters? Maybe my email address is ``am@gmail.com`` (it's a valid one).

Reply:
@amihaiemil  `f"@{host}"` is the same as `"@" + host`. welcome to python world :)
I'll add length checks today :+1: